### PR TITLE
Update rustfmt max width to 80

### PR DESCRIPTION
**Pull request by alanssitis**
From the following [comment](https://github.com/in-toto/in-toto-rs/pull/70#discussion_r1347606569). Updated width to 80.

Based on #70.

A clippy warning also came up, which was resolved here as well.

### CODE REVIEW

**Comment by adityasaky** [comment]
Can you add a commit with a file like https://github.com/in-toto/in-toto/blob/develop/.git-blame-ignore-revs referencing the commit that fixes line lengths across the code base?

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Ignore calls to `cargo fmt`
+#
+# To learn more, see
+#     https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view
+
+# Cargo fmt changing max width to 80
+94b7757698baa135a2f2139f3669af10f7e71e60
```
Comment (adityasaky) [inline .git-blame-ignore-revs:7]: I'm guessing you're still working through this. If I'm not mistaken, I currently don't see this commit in this PR? Also, I recommend:
* Commit 1: Make config change
* Commit 2: Reformat the code base
* Commit 3: Introduce the ignore-revs file with commit 2

Comment (alanssitis) [reply .git-blame-ignore-revs]: Yeah, I think I messed up a bit when copying it.

Done now though.

**Review by adityasaky — APPROVED** [review]
Thanks!

Confirmed: tree ID aligns when I run `cargo fmt` with that of c3459b1715fe488ab890ed2901d3486d6ebf2fa0 locally, after the config update commit.

**Pull request merged**